winch/codegen/src/trampoline.rs

```diff
@@ -601,6 +528,9 @@ where
         callee_sig
             .params_without_retptr()
             .iter()
+            // Skip the first two arguments, which are the callee and caller
+            // VMContext pointers.
+            .skip(2)
```

Comment:
here and perhaps elsewhere, can we have a constant (maybe an associated constant of `ContextArgs`) indicating how many args we expect? Basically it'd be nice to tie this to other definitions so that we don't miss it if we ever add, e.g., a third context arg.